Probesys/bileto
# new: Turn ticket into incident or request

Related to https://github.com/Probesys/bileto/issues/33

Changes proposed in this pull request:

- Add an "Actions" menu to the ticket page
- Add a menu item to switch between incident and request
- Change how type is displayed to handle "cet incident" vs. "cette demande" in French

How to test the feature manually:

1. Open a ticket
2. Open the actions menu and turn the ticket into an incident
3. Check the page now displays that the ticket is an incident

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
